packages/a/arp-scan/pspec_x86_64.xml

```diff
@@ -20,27 +20,25 @@
 </Description>
         <PartOf>network.util</PartOf>
         <Files>
+            <Path fileType="config">/etc/arp-scan/mac-vendor.txt</Path>
```

Comment:
Because we try to be Stateless, packages shouldn't ship files in `/etc`. It looks like it was in `/usr/share/arp-scan/` before. Can we move it back there?